modules/ingester/config.go

```diff
@@ -48,6 +48,7 @@ func (cfg *Config) RegisterFlagsAndApplyDefaults(prefix string, f *flag.FlagSet)
 	f.DurationVar(&cfg.MaxBlockDuration, prefix+".max-block-duration", 30*time.Minute, "Maximum duration which the head block can be appended to before cutting it.")
 	f.Uint64Var(&cfg.MaxBlockBytes, prefix+".max-block-bytes", 500*1024*1024, "Maximum size of the head block before cutting it.")
 	f.DurationVar(&cfg.CompleteBlockTimeout, prefix+".complete-block-timeout", 3*tempodb.DefaultBlocklistPoll, "Duration to keep blocks in the ingester after they have been flushed.")
+	f.StringVar(&cfg.LifecyclerConfig.Zone, prefix+".availability-zone", "default-zone", "Define Availability Zone in which this ingester is running in")
```

Comment:
this is currently defaulting to empty string. i'm concerned about changing it b/c i don't totally understand the impact.

any reason not to default to empty string?

Reply:
changed it back to empty string